tests/test_annots.py

```diff
@@ -158,8 +159,8 @@ def test_redact1():
         rotate=-1,
     )
     assert annot.type == (12, "Redact")
-    x = annot._get_redact_values()
-    pix = annot.get_pixmap()
+    annot._get_redact_values()
+    annot.get_pixmap()
```

Comment:
It's not that important, but this test is calling internal implementation method `annot._get_redact_values()`, which seems a bit unusual. Could the test be changed to only use the public API?